README.md

```diff
@@ -17,11 +15,27 @@ The package is intended to simplify the process of testing application elements
 [![static analysis](https://github.com/yiisoft/test-support/workflows/static%20analysis/badge.svg)](https://github.com/yiisoft/test-support/actions?query=workflow%3A%22static+analysis%22)
 [![type-coverage](https://shepherd.dev/github/yiisoft/test-support/coverage.svg)](https://shepherd.dev/github/yiisoft/test-support)
 
-### Documentation
+The package is intended to simplify the process of testing application elements that depend on PSR interfaces.
+
+## Requirements
+
+- PHP 7.4 or higher.
+
+## Installation
+
+The package could be installed with composer:
+
+```shell
+composer require yiisoft/test-support --prefer-dist
+```
```

Comment:
Please read the documentation for this package.

Reply:
Added `--dev`